spec/openapi.yaml

```diff
@@ -483,6 +483,42 @@ paths:
         409:
           $ref: "#/components/responses/Conflict"
           description: If the file to be deleted is currently being printed
+  /api/system/commands/core/changeexposure:
```

Comment:
Please replace new endpints next to https://github.com/prusa3d/Prusa-Connect-Local/blob/master/spec/openapi.yaml#L225

Reply:
I would also change is to custom command and add `default` for it